scripts/test.py

```diff
@@ -0,0 +1,97 @@
+import os
+import re
+import unittest
+
+def model(model_name):
+  return '--model=../model_zoo/historical/prototext/%s.prototext' % model_name
+
+def reader(reader_name):
+  return '--reader=../model_zoo/data_readers/%s.prototext' % reader_name
+
+def optimizer(optimizer_name):
+  return '--optimizer=../model_zoo/optimizers/%s.prototext' % optimizer_name
+
+def executable(cluster):
+  return '../build/%s/model_zoo/lbann' % cluster
+
+def run_lbann(model_name, reader_name, optimizer_name, output_file_name):
+  if (model_name == None or reader_name == None or optimizer_name == None or output_file_name == None):
+    raise Exception("Invalid number of arguments")
+  else:
+    os.system('hostname > %s' % output_file_name)
+    hostname_file = open(output_file_name, 'r')
+    hostname = hostname_file.readline()
+    hostname_file.close()
+    cluster = re.sub('[0-9]+\n', '.llnl.gov', hostname)
+    e = executable(cluster)
+    m = model(model_name)
+    r = reader(reader_name)
+    o = optimizer(optimizer_name)
+    # N => number of nodes                                                         
+    # p => partition
+    # t => timeout period, in minutes
+    # n => number of processes to run. MPI Rank. How many models being made.
+    # num-epochs => The number of epochs
+    os.system('salloc -N 1 -p pbatch -t 30 srun -n 2 %s %s %s %s --num_epochs=5 > %s' % (e, m, r, o, output_file_name))
```

Comment:
Remove `-t 30` for longer running times? Are there any options besides model, reader, optimizer we want to specify?

Reply:
Eventually we are going to want to vary the number of nodes and processes, which also dictates the processes per node.